common-react/@dbeaver/react-data-grid/src/renderers/HeaderCellContentRenderer.tsx

```diff
@@ -1,13 +1,37 @@
-import { use } from 'react';
+import { use, useRef } from 'react';
 import { DataGridCellHeaderContext } from '../DataGridHeaderCellContext.js';
 import { useGridReactiveValue } from '../useGridReactiveValue.js';
+import { OrderButton } from './OrderButton.js';
 
 interface Props {
   colIdx: number;
+  tabIndex?: number;
 }
-export function HeaderCellContentRenderer({ colIdx }: Props) {
+export function HeaderCellContentRenderer({ colIdx, tabIndex }: Props) {
+  const attributePosition = colIdx - 1;
   const cellHeaderContext = use(DataGridCellHeaderContext);
   const headerElement = useGridReactiveValue(cellHeaderContext?.headerElement, colIdx);
   const getHeaderText = useGridReactiveValue(headerElement ? undefined : cellHeaderContext?.headerText, colIdx);
-  return headerElement ?? getHeaderText ?? '';
+  const isColumnSortable = cellHeaderContext?.getColumnSortable?.(colIdx);
+  const onColumnSort = cellHeaderContext?.onColumnSort;
+  const sortingState = cellHeaderContext?.getColumnSortingState?.(attributePosition);
```

Comment:
Probably we need to use reactive value for the `getColumnSortable ` and `getColumnSortingState` (and rename them: remove get like in other props for reactive value) because it may not update in some cases, due. to how MobX works